gr-uhd/lib/usrp_source_impl.cc

```diff
@@ -449,7 +451,12 @@ void usrp_source_impl::set_stream_args(const ::uhd::stream_args_t& stream_args)
     }
 }
 
-void usrp_source_impl::_cmd_handler_tag(const pmt::pmt_t& tag) { _tag_now = true; }
+void usrp_source_impl::_cmd_handler_tag(const pmt::pmt_t& tag,
+                                        int chan,
+                                        const pmt::pmt_t& msg)
+{
```

Comment:
```suggestion
void usrp_source_impl::_cmd_handler_tag(const pmt::pmt_t&)
{
```

Reply:
Done, the latest push keep signature of ``_cmd_handler_tag()`` unchanged.